src/org/zaproxy/zap/extension/bruteforce/BruteForceParam.java

```diff
@@ -173,4 +186,59 @@ public void setFileExtensions(String fileExtensions) {
 	    
     	return fileExtensionsList;	    
 	}
+
+	/**
+	 * @return {@code String} of comma-separated file-extensions that are ignored.
+	 * {@link #DEFAULT_EXTENSIONS_TO_MISS} is returned by default
+	 */
+	public String getExtensionsToMiss() {
+		return extensionsToMiss;
+	}
+
+	/**
+	 * Define a {@code String} of comma-separated file-extensions for 
+	 * resources to ignore
+	 * 
+	 * @param extensionsToMiss file-extensions string
+	 * @throws IllegalArgumentException if {@code extensionsToMiss} is
+	 * {@code null}
+	 */
+	public void setExtensionsToMiss(String extensionsToMiss) {
+		if (extensionsToMiss == null) {			
+			throw new IllegalArgumentException("extensionsToMiss is null");
+		} 
+		
+		this.extensionsToMiss = extensionsToMiss;
+		getConfig().setProperty(EXTENSIONS_TO_MISS, extensionsToMiss);
+	}
+
+	/**
+	 * @return {@code Set} of file extensions that are ignored.
+	 * By default returned {@code Set} will contain following extensions, 
+	 * {@link #DEFAULT_EXTENSIONS_TO_MISS}
+	 */
+	public Set<String> getExtensionsToMissSet() {
+		if (extensionsToMiss.trim().equals(EMPTY_STRING)) {
+	    	return new HashSet<>();
+	    }
+		String[] tempArray = extensionsToMiss.replaceAll("\\s", 
+	    		EMPTY_STRING).split(",");
+		Set<String> tempSet = new HashSet<>(Arrays.asList(tempArray));
+		tempSet.remove(EMPTY_STRING);
+		return tempSet; 
+	}
+
+	public String getFailCaseString() {
+		return failCaseString;
+	}
+
+	public void setFailCaseString(String failCaseString) {
+		if (failCaseString == null || failCaseString.isEmpty()) {
+	    	throw new IllegalArgumentException(
+	    			Constant.messages.getString("bruteforce.options.error.failCaseString.invalid"));

```

Comment:
For exception messages not show directly in the UI we keep them in English.
The `failCaseString` is not being set to the configuration file.